docs/_documentations/remotedeploy-single.md

```diff
@@ -10,58 +10,58 @@ parent: installoncloud
 order: 2
 ---
 
-# Installing a Codewind remote deployment
+# Installing components individually for a Codewind remote deployment
 
-Codewind includes a CLI to simplify the installation process. You will find `cwctl` in your HOME directory under the path ~/.codewind/{version}
+Codewind includes a CLI to simplify the installation process. You can find the `cwctl` CLI in your HOME directory under the path `~/.codewind/{version}`.
 
-The command for installing an "all-in" deployment with a new Keycloak and new Codewind service is :
+The command for installing components individually for a remote deployment with a new Keycloak and a new Codewind service is:
 
 `cwctl --insecure install remote` 
 
-and requires various flags to specify where and what to install.
+This command requires various flags to specify where and what to install.
 
 ## Deploying Keycloak 
 
-1.  Open a new terminal window on your local workstation
-2.  Navigate to your home directory and to the Codewind cli:
+1.  Open a new terminal window on your local workstation.
+2.  Navigate to your home directory, and then to the Codewind CLI:
 
 ```
 cd ~/.codewind/0.8.0
 ```
 
-Ensure you are logged into your Kubernetes or Openshift cluster
+Ensure that you are logged into your Kubernetes or Openshift cluster:
 
 ```
 $ kubectl get namespaces
 ```
 
-If the command is successful you should see a list of current namespaces.  If not,  ensure you are logged into your Kubernetes or Openshift cluster.
+If the command is successful, you see a list of current namespaces. If not, ensure that you are logged into your Kubernetes or Openshift cluster.
 
 ## Determine your Cloud ingress domain
 
-The cli command requires an ingress domain,  you can find yours based on any of the existing routes:
+The CLI command requires an ingress domain, you can find yours based on any of the existing routes:
```

Comment:
This sentence should be two: "The CLI command requires an ingress domain. Find yours based on any of the existing routes:"